actions.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+type DataInterface interface{}
+
+func GetActionFromType(commandType string) Action {
+	var data DataInterface
+	switch commandType {
+	case "message":
+		data = MessageData{}
+	case "help":
+		data = HelpData{}
+	case "update":
+		data = UpdateData{}
+	case "lookingFor":
+		data = LookingForData{}
+	case "notLookingFor":
+		data = NotLookingForData{}
+	case "yearly":
+		data = YearlyData{}
+	case "todayLectures":
+		data = TodayLecturesData{}
+	case "tomorrowLectures":
+		data = TomorrowLecturesData{}
+	case "list":
+		data = ListData{}
+	case "course":
+		data = CourseData{}
+	case "luck":
+		data = LuckData{}
+	default:
+		data = MessageData{}
+	}
+
+	return Action{
+		Name: "UNINITIALIZED",
```

Comment:
Visto come funziona l'unmarshaling di Go per JSON è piuttosto probabile che @crestaa vorrà cambiare questa funzione in futuro, così ne approfittiamo anche per evitare la pratica di avere variabili pseudo-inizializzate. Per ora però direi che può andar bene.

Reply:
(Quindi questa la segno in autonomia come risolta: era più per fare una previsione.)

Reply:
Questo ha senso, potrei aggiugnere una nuova struct invalida, e prendere come input anche il nome per inizializzarlo.